### PR TITLE
BWE tweaks

### PR DESCRIPTION
**Pull request by k0nserv**
- Don't use queues for padding before first media
- Increase max consecutive padding packets
- Tweak pacing and padding rates

### CODE REVIEW

File: src/media/inner.rs [hunk 1]
```diff
@@ -1128,7 +1128,9 @@ impl MediaInner {
         let state = QueueState {
             mid: self.mid,
             is_audio: self.kind.is_audio(),
-            use_for_padding: self.kind.is_video() && self.has_tx_rtx(),
+            use_for_padding: self.kind.is_video()
+                && self.has_tx_rtx()
+                && snapshot.last_emitted.is_some(),
```
Comment (algesten) [inline src/media/inner.rs:1133]: What is the semantic meaning of this third argument?

File: src/packet/pacer.rs [hunk 2]
```diff
@@ -11,7 +11,7 @@ const MAX_BITRATE: Bitrate = Bitrate::gbps(10);
 const MAX_DEBT_IN_TIME: Duration = Duration::from_millis(500);
 const MAX_PADDING_PACKET_SIZE: DataSize = DataSize::bytes(224);
 const PADDING_BURST_INTERVAL: Duration = Duration::from_millis(5);
-const MAX_CONSECUTIVE_PADDING_PACKETS: usize = 5_000;
+const MAX_CONSECUTIVE_PADDING_PACKETS: usize = 1 << 16;
```
Comment (algesten) [inline src/packet/pacer.rs:14]: u16::max ?

**Review by algesten — COMMENTED** [review]
Some small things.

I like the cleanup in Session!

File: src/media/inner.rs (hunk 1 above)
Comment (k0nserv) [reply src/media/inner.rs:1133]: It's saying that we only use queues for padding if we have previously sent something on them, definitionally that thing will be media

Comment (algesten) [reply src/media/inner.rs:1133]: Could you maybe make that clearer by having `snapshot.has_sent_something()` or similar?

Comment (k0nserv) [reply src/media/inner.rs:1133]: Sure

**Review by algesten — APPROVED** [review]
LGTM

**Review by davibe — APPROVED** [review]

**Pull request merged**